recorder.go

```diff
@@ -38,6 +45,11 @@ func (rr *RequestRecorder) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if r.URL.Path == "/gopherman-reset" {
```

Comment:
Let's just reset the collection in the terminate handler, rather than having it be a separate handler.

Reply:
I'd prefer to have them separate. I've found it to be handy to be able to check on the requests, as you are building up to a more complex tasks. 

If you want terminate to include reset, then how about adding a third endpoint that does not reset, like `gopherman-current`?

Reply:
Aka swap the endpoint roles.

Reply:
Alright I get your reasoning. This can stay as is.